### DISCUSSION TITLE
ESP32-C3-SuperMini

### DISCUSSION THREAD

**Discussion by BritishTechGuru** [espressif/arduino-esp32#9226] [Q&A]
I just bought one from Temu and can't get code into it.
I tried it with Thonny but Thonny rejected the Github Micropython for the C3, saying it was the wrong chip. 
I then tried uploading the blink program from the Arduino IDE and it got to a message where it said it was hard resetting the board. Then the IDE just stalled and the board now has a red and blue LED on but no blinking.

I put it aside now. If a solution comes along, fine. Until then I'll work with my other boards..

**Comment by DevZillion** [comment]
Added by #9050
